.eslintignore

```diff
@@ -0,0 +1,6 @@
+dist/*
+build/*
+
+test/jasmine/assets/jquery-1.8.3.min.js
+src/plots/polar/micropolar.js
+src/plots/geo/projections.js
```

Comment:
We should add `node_modules` in here too, no?

I see that projections is a forked file from somewhere else, but why ignore micropolar?